exampleCourse/questions/element/hiddenHints/question.html

```diff
@@ -13,8 +13,13 @@
 
 <pl-question-panel>
   <pl-hidden-hints>
+    <!-- Hints with the same show-after-submission appear in the order they're written -->
     <pl-hint>
-      This is a hint that will be shown immediately.
+      Another hint that will be shown immediately.
```

Comment:
Nit: the "another" one should probably come second?

Reply:
This is actually on purpose to test the behavior is as-expected. "Another" is before "A " lexicographically (and I couldn't think of a generic hint message that started with the letter "B" 😅 )

Reply:
Hah, fair! If you'd like to test behavior, consider writing a unit test instead.

Reply:
I think I probably will if I need to make any more complex changes to this element. Unfortunately hard to strip out the ordering logic from parsing the element itself (and I don't want to move in all of our test fixtures just for testing something this simple, they're pretty huge).

Reply:
You can and should write Python tests for elements (at least for the `server.py` code). And while we don't have any tests that test a rull render yet, it shouldn't be hard to add: you can just pass in a blob of HTML to the `render` function and make assertions about the returned HTML.

Feel free to save that for another PR thought - I'll merge this as-is!